src/main/webapp/index.html

```diff
@@ -33,7 +33,7 @@ <h1>CodeU Starter Project</h1>
     <p>This is the CodeU starter project. Click the links above to login and visit your page.
     You can post messages on your page, and you can visit other user pages if you have
     their URL.</p>
-    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
-    You could start by modifying this page to tell the world more about your team.</p>
```

Comment:
intentional change?

Reply:
This was me trying to figure out what they wanted for index.html changes for week0... should i change this back?

Reply:
Yeah lets keep the info on the about us page